scene/resources/font.cpp

```diff
@@ -1118,11 +1118,11 @@ bool FontFile::_set(const StringName &p_name, const Variant &p_value) {
 #endif // DISABLE_DEPRECATED
 
 	if (tokens.size() == 2 && tokens[0] == "language_support_override") {
-		String lang_code = tokens[1];
+		const String &lang_code = tokens[1];
 		set_language_support_override(lang_code, p_value);
 		return true;
 	} else if (tokens.size() == 2 && tokens[0] == "script_support_override") {
-		String script_code = tokens[1];
+		const String &script_code = tokens[1];
 		set_script_support_override(script_code, p_value);
```

Comment:
```suggestion
		set_script_support_override(tokens[1], p_value);
```

Reply:
I'm against this change since I think there's context loss as to what `tokens[1]` mean, which in this case is the Script Code of the font. Without the assignment, I don't think that's clear.